packages/backend/src/persistance/getId.ts

```diff
@@ -0,0 +1,13 @@
+import { Redis, RedisPrefix } from './getDatabase';
+import * as Lib from 'dm3-lib/dist.backend';
+
+export function getId(redis: Redis) {
```

Comment:
I think getId can be part of the Session package since it references the Session prefix

Reply:
Yes, I changed it